src/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+
+
+interface ButtonProps {
+  buttonText: string;
+  link: string;
+}
+
+export const Button: React.FC<ButtonProps> = ({ buttonText, link }) => {
```

Comment:
I suggest to rename component. Button is a general name. in this case for example maybe call like NavButton